File: Python PowerPoint Examples/SlidesApiDemo/reorderManySlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "SlidesReorderMany_1.pptx"
password = ""
storage = ""
old_positions = "5,1,0"
new_positions = "0,2,3"
inputFolder = "input"
result =api.reorder_many_slide(fileName, old_positions=old_positions, new_positions=new_positions, password=password,storage=storage, folder=inputFolder)

File: JavaScript PowerPoint Examples/SlidesApiDemo/reorderManySlide.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.SlidesApi(apiClient);

    var name = "reorderManySlide.pptx";
    var folder = "input";
    //If there isn't password, just remove the corresponding parameters
    var opt = { 'oldPositions': '0,1', 'newPositions': '1,0', 'password': '123', 'folder': folder};
    var response = instance.reorderManySlide(name, opt,
        function (error, data, response) {
            if (error) throw error;
    });
})();